tensorboard/webapp/app_routing/actions/app_routing_actions.ts

```diff
@@ -70,6 +70,10 @@ export const navigating = createAction(
 export interface NavigatedPayload {
   before: Route | null;
   after: Route;
+  // TODO(bdubois): Remove optionality of these properties when all internal
+  // code is migrated to specify them.
+  beforeNamespaceId?: string | null;
```

Comment:
Two new properties in navigated() payload. Temporarily they are optional to allow internal build to pass.